pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -124,6 +125,8 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	operationGenerator OperationGenerator) OperationExecutor {
 
+	util.RegisterMetrics()
```

Comment:
I just copied others, it looks like a sort of unspoken convention to use sync to prevent multiple registration https://github.com/kubernetes/kubernetes/search?utf8=%E2%9C%93&q=sync.once+registermetrics&type=